Source/PagesController.swift

```diff
@@ -56,7 +81,7 @@ extension PagesController {
         completion: { [unowned self] finished in
           self.pagesDelegate?.pageViewController(self,
             setViewController: viewController,
-            atPage: currentIndex)
+            atPage: self.currentIndex)
```

Comment:
@kostiakoval been in a closure isn't the only requirement, you also need to have another variable with the same name before having to use `self`.















I might be wrong.